fenix/app/src/main/java/org/mozilla/fenix/debugsettings/tabs/TabTools.kt

```diff
@@ -161,6 +194,90 @@ private fun TabCountRow(
     }
 }
 
+private const val DEFAULT_TABS_TO_ADD = "1"
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+private fun TabCreationTool(
+    onAddTabsToActiveClick: ((Int) -> Unit),
+    onAddTabsToInactiveClick: ((Int) -> Unit),
+    onAddTabsToPrivateClick: ((Int) -> Unit),
+) {
+    var tabQuantityToCreate by rememberSaveable { mutableStateOf(DEFAULT_TABS_TO_ADD) }
+    var hasError by rememberSaveable { mutableStateOf(false) }
+    val keyboardController = LocalSoftwareKeyboardController.current
+
+    Column {
+        Text(
+            text = stringResource(R.string.debug_drawer_tab_tools_tab_creation_tool_title),
+            color = FirefoxTheme.colors.textPrimary,
+            style = FirefoxTheme.typography.headline5,
+        )
+
+        TextField(
```

Comment:
I'm going to file a follow-up after this lands to make this TextField a top-level component. 

The colors were all derived from the [figma](https://www.figma.com/file/ujdiReypSypu2GhRgzArea/Android-Components?type=design&node-id=3554-41479&mode=design&t=1tVWzjhdtyjyK59q-0)

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1871470

Reply:
This seems like a good idea! I honestly don't know much about handling `Keyboard` interactions correctly, so I'm just wondering if we're handling everything we need to from an accessibility standpoint

Reply:
For the purposes of this text field, we should be all set. When we stand up a `TextField` composable though, we'll need to create the correct API to capture/document all of the `Keyboard` interactions.